teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < person
+  def initialize(_specilization, age, name = 'Unknown', parent_pemission: true)
+    super age, name, parent_pemission
+    @specialization = specialization
```

Comment:
- [ ] Kindly change the `@specialization = specialization` to `@specialization = _specialization` as the name of the parameter is `_specialization` OR you can change the parameter to `specialization` and leave it as it's.